### ISSUE TITLE
Filename with multiple dots creates problems

### ISSUE THREAD

**Issue by m1ga** [yomybaby/atom-titanium#62]
Setup:

```bash
ti create -t app -n test --id com.test.test -p all -u test.de -d .
cd test
alloy new
alloy generate controller test/test.file.name
atom .
```

Now if you open `app/views/test/test.file.name.xml` and Ctrl+Click on the class it will create a new class called `app/styles/test/test.tss.name.xml` instead of opening the correct `app/styles/test/test.file.name.tss`

**Comment by yomybaby** [comment]
Thank you for reporting. I'll check this.

**Comment by yomybaby** [comment]
@m1ga  I just fixed #61 #62 .
Please check these issues on v0.13.9.
Thanks for your reporting with detail description.🙏🏼

**Comment by m1ga** [comment]
awesome, will test it tomorrow! Thank you
